app/services/feature_flag.rb

```diff
@@ -3,6 +3,7 @@ class FeatureFlag
     rate_limiting
     display_sign_in_token_links
     show_component_previews
+    gias_data_stage_pause
```

Comment:
Looking at the FeatureFlag code, a feature is only active if their associated envar has the value *'active'*, otherwise it won't be active.
So we don't strictly need it but are you planning to set the new envar to *'inactive'* manually in all enviroments via the cf command line?

Reply:
Ideally we should set it to `inactive`, but I purposely made it so we set it when we need it and without it, it's not going to break anything. 

I will in the other ticket do a clean up and make sure they are all set to something and remove the unused ones :)

Reply:
Great! LGTM